app/src/main/java/nextstep/shoppingcart/view/resource/Dimens.kt

```diff
@@ -0,0 +1,12 @@
+package nextstep.shoppingcart.view.resource
+
+import androidx.compose.ui.unit.dp
+
+val GridStartPadding = 18.dp
+val GridEndPadding = 18.dp
+val GridTopPadding = 13.dp
+val ProductNameBottomPadding = 2.dp
+val GridHorizontalSpace = 12.dp
+val GridVerticalSpace = 20.dp
+const val GridCellsCount = 2
+const val ProductNameMaxLine = 1
```

Comment:
ui.theme의 경우도 XML의 리소스가 할 역할을 온전히 대신하지는 못합니다.

참고로 Color의 경우는 다음과 같이 설명되어 있습니다.
참고: https://developer.android.com/develop/ui/compose/resources#colors
> Warning: Prefer colors from the theme rather than hard-coded colors. Even though it's possible to access colors using the colorResource function, it's recommended that the colors of your app are defined in a MaterialTheme which can be accessed from your composables like MaterialTheme.colors.primary. Learn more about this in the [Design systems in Compose documentation](https://developer.android.com/develop/ui/compose/designsystems).